package-lock.json

```diff
@@ -1397,12 +1406,20 @@
       "dev": true
     },
     "async": {
-      "version": "2.6.1",
-      "resolved": "https://registry.npmjs.org/async/-/async-2.6.1.tgz",
-      "integrity": "sha512-fNEiL2+AZt6AlAw/29Cr0UDe4sRAHCpEHh54WMz+Bb7QfNcFw4h3loofyJpLeQs4Yx7yuqu/2dLgM5hKOs6HlQ==",
+      "version": "2.6.2",
```

Comment:
Did you run some other npm updates here? Not sure why this package was bumped since it doesn't seem to be a dependency of cdk.

Reply:
NPM seems to update packages on its own despite running npm install only. Should I revert this?

Reply:
Yes, please only make changes that are necessary for this PR.